### ISSUE TITLE
One single source of version information

### ISSUE THREAD

**Issue by atilag** [QISKit/qiskit-core#581]
### What is the expected enhancement?
Currently, there are three places where we maintain Version number information:
* CMakeFiles.txt
* qiskit/\_\_init\_\_.py
* setup.py.in

Every time we have to release a new version, we have to change `CMakeFiles.txt` and `qiskit/__init__.py` files, and make sure both of them have the same number. The CMake system will update the last one: `setup.py.in`. So there's no need to change anything there yet.

We want to have only one source for our versioning system, and the proposed mechanism is the fourth point of this document: https://packaging.python.org/guides/single-sourcing-package-version/
```
4. Place the value in a simple VERSION text file and have both setup.py and the project code read it.
```

Tasks
=====
* Modify CMakeFiles.txt to remove the Version information from the `project(...)` function.
* Add a mechanism to load the version information from an external file in CMake, and append it to
  the QISKIT_VERSION variable.
* Change qiskit/__init__.py to load the version information from an external file.
* Add the external version information file to the MANIFEST.in file

How to test
=========
Starting from a clean project directory (no setup.py file present). Just build with cmake [1]:
* mkdir out
* cd out
* cmake ..
* See that the setup.py file auto-generated after the `cmake ..` command  was executed, has the correct information number.

[1] More information on how to build in the CONTRIBUTING guide.

**Comment by chrisculver** [comment]
I just finished this and am submitting a pull request.  

Is there a way to assign myself to issues in the future?

**Comment by ajavadia** [comment]
Closed in #581. Thanks @chrisculver !
